tox.ini

```diff
@@ -66,4 +66,4 @@ skip_install = true
 description = check code style
 deps = flake8
 changedir = {toxinidir}
-commands = flake8 pyvo --count
+commands = flake8 pyvo --count --ignore=E128,E131,W503
```

Comment:
Don't do it this way, but add them in the ignore list in setup.cfg, so there will be no errors for local flake8 runs either.
Nitpick: I think it's good to logically separate commit contents (e.g. with your other recent PR it would have been a bit easier to just cherry-pick the test commit for regression testing on main), but this is very much down to personal preference, and I'm not one to enforce my preference (unless a PR incrementally adds zillions of commits with irrelevant contents for e.g. code debug, or add and remove files, none of which is really an issue here in pyvo)